third_party/gpus/find_cuda_config.py

```diff
@@ -65,10 +65,7 @@
 import sys
 
 # pylint: disable=g-import-not-at-top
-try:
-  from shutil import which
-except ImportError:
-  from distutils.spawn import find_executable as which
+from shutil import which
 # pylint: enable=g-import-not-at-top
```

Comment:
Linter complains (here and in other places where this pattern occurs): Since now we have a top-level import, it should be sorted in the list of system imports that come above it. Also, the pylint comments are no longer needed.

Can you do these changes please?

Reply:
Okay, I've used `shutil.which` instead of `which` to be more consistent with the rest of the imports.